incubator/hnc/pkg/reconcilers/hnc_config_test.go

```diff
@@ -284,6 +283,30 @@ var _ = Describe("HNCConfiguration", func() {
 		Eventually(hasObject(ctx, "CronTab", barName, "foo-crontab")).Should(BeTrue())
 		Expect(objectInheritedFrom(ctx, "CronTab", barName, "foo-crontab")).Should(Equal(fooName))
 	})
+
+	It("should set NumPropagatedObjects back to 0 after deleting the source object in propagate mode", func() {
```

Comment:
This also checks that the propagated count _increases_ to non-zero, right?

Reply:
Yes. That's why it says `back to 0`. Feel free to let me know if I need to make the description more clear. I am heading off now but I can update it in a separate PR.